engine/src/flutter/shell/platform/android/io/flutter/plugin/platform/PlatformPlugin.java

```diff
@@ -393,7 +408,13 @@ private void setSystemChromeEnabledSystemUIOverlays(
    * PlatformPlugin}.
    */
   public void updateSystemUiOverlays() {
-    activity.getWindow().getDecorView().setSystemUiVisibility(mEnabledOverlays);
+    // Use the WindowCompat to set Edge-To-Edge.  Call this even if not Edge-To-Edge in case the
+    // flags need to be reset. If Edge-To-Edge, this method should be passed false.
```

Comment:
Consider a link to https://developer.android.com/develop/ui/views/layout/edge-to-edge-manually as a breadcrumb.